django/db/backends/base/base.py

```diff
@@ -98,6 +98,15 @@ def __init__(self, settings_dict, alias=DEFAULT_DB_ALIAS,
         self.introspection = self.introspection_class(self)
         self.ops = self.ops_class(self)
         self.validation = self.validation_class(self)
+        # A stack of wrappers to be invoked around execute()/executemany()
+        # calls.
+        #
+        # Each entry is a (wrapper, for_many) tuple, where wrapper is a
```

Comment:
Is this comment still accurate regarding `for_many`?

Reply:
🦅 eagle eye award

Reply:
Thanks, fixed.